client/assets/calendar.js

```diff
@@ -0,0 +1,121 @@
+import { saveAs } from 'file-saver'
+import Moment from 'moment'
+import {
+  extendMoment
+} from 'moment-range'
+
+export async function downloadCalendar({ plan }) {
+	const FileSaver = require('file-saver')
+	const moment = extendMoment(Moment)
```

Comment:
This is constant so it should be hoisted up outside this function so it doesn't get created every time we download (not that it is that costly just out of good practice hoist all constants above the function so the function can do its work.

Reply:
Did you mean for this to be about downloadPlanner in planner.js? Because this is the same function as the comment below